paltas/Configs/config_handler.py

```diff
@@ -506,8 +506,16 @@ def _draw_image_standard(self,add_noise=True,apply_psf=True):
 			if mag < self.mag_cut:
 				raise MagnificationError(self.mag_cut)
 
+		# Add constant sky brightness
+		# Note lenstronomy's sky brightness is in counts/square arcsec, 
+		# so to get the value per pixel, we have to multiply by resolution^2
+		image += single_band._sky_brightness_cps * kwargs_detector['pixel_scale']**2
```

Comment:
If I understand this change correctly, it now adds a constant background even when there is no noise. I believe the noise_for_model function doesn't add this background which leads me to a few comments:

1. Should we be using this internal property of single_band or the public function https://github.com/lenstronomy/lenstronomy/blob/67949f01fa26f70aa551fef9fb853ce4c0baef20/lenstronomy/SimulationAPI/observation_api.py#L221 that accounts for ADU
2. I think we should make this an explicit property of the config instead of changing the default behavior. Maybe something "add_overall_offset" or whatever other creative name comes to mind.
3. I think setting _sky_brithness_cps to 0 is a bug, since then it won't include the noise from the sky brightness. It would be good to test this behavior explicitly to make sure it's doing what you want.